packages/contracts/src/plugins/governance/admin/build-metadata.json

```diff
@@ -1,9 +1,21 @@
 {
   "ui": "",
-  "change": "",
+  "change": "This is the first build of the first release.",
   "pluginSetupABI": {
-    "prepareInstallation": ["address admin"],
-    "prepareUpdate": [""],
-    "prepareUninstallation": [""]
+    "prepareInstallation": {
+      "description": "The information required for the installation of build 1.",
```

Comment:
```suggestion
      "description": "Installs a new plugin that allows the given address to execute proposals on the DAO",
```

Reply:
This seems to be more of a general description of what the Admin plugin does. It doesn't really describe the installation process `prepareInstallation` itself.